### PR TITLE
Update user-consent.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the navigation to M365 admin center and also added a link to the portal.

### CODE REVIEW

**Comment by msbemba** [comment]
per issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10128

@JohanFreelancer9 Please copy review

File: microsoft-365/admin/misc/user-consent.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ A user can give access only to apps they own that access their Office 365 inform
 
 Here's how to turn User consent to apps on or off.
 
-1. In the admin center, go to the **Settings** \> **Org settings** > [Services](https://go.microsoft.com/fwlink/p/?linkid=2053743) page, and then select **User consent to apps**.
+1. In the  [M365 admin center](https://admin.microsoft.com), go to the **Settings** \> **Org settings** > [Services](https://go.microsoft.com/fwlink/p/?linkid=2053743) page, and then select **User consent to apps**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/user-consent.md]:
```suggestion
1. In the  [M365 admin center](https://admin.microsoft.com), go to the **Settings** > **Org settings** > [Services](https://go.microsoft.com/fwlink/p/?linkid=2053743) page, and then select **User consent to apps**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Review by kwekuako — COMMENTED** [review]
Can you update line 41 and resubmit? We don't write M365 admin center. Spell it out fully: Microsoft 365 admin center or if its the second use or more, we can write "admin center"

File: microsoft-365/admin/misc/user-consent.md [hunk 2]
```diff
@@ -38,11 +38,11 @@ A user can give access only to apps they own that access their Office 365 inform
 
 Here's how to turn User consent to apps on or off.
 
-1. In the admin center, go to the **Settings** \> **Org settings** > [Services](https://go.microsoft.com/fwlink/p/?linkid=2053743) page, and then select **User consent to apps**.
+1. In the  [M365 admin center](https://admin.microsoft.com), go to the **Settings** > **Org settings** > [Services](https://go.microsoft.com/fwlink/p/?linkid=2053743) page, and then select **User consent to apps**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/user-consent.md]:
```suggestion
1. In the  [Microsoft 365 admin center](https://admin.microsoft.com), go to the **Settings** > **Org settings** > [Services](https://go.microsoft.com/fwlink/p/?linkid=2053743) page, and then select **User consent to apps**.
```

**Comment by msbemba** [comment]
@kwekuako  We have updated the PR now.

**Pull request merged**